src/components/DataTable/table.scss

```diff
@@ -46,8 +46,11 @@ thead .data-table-tr th {
 
     & .data-table-tr {
         height: 48px;
-        & .data-table-th {
```

Comment:
I deleted this style because it's (auto) the wrong value for padding. This style doesn't work. I tested it, it works exactly the same without it